Jellyfin.Api/Controllers/SystemController.cs

```diff
@@ -71,6 +71,19 @@ public SystemController(
     public ActionResult<SystemInfo> GetSystemInfo()
         => _systemManager.GetSystemInfo(Request);
 
+    /// <summary>
+    /// Gets information about the server.
+    /// </summary>
+    /// <response code="200">Information retrieved.</response>
+    /// <response code="403">User does not have permission to retrieve information.</response>
+    /// <returns>A <see cref="SystemInfo"/> with info about the system.</returns>
+    [HttpGet("StorageResources")]
```

Comment:
I would remove the "storage" part from the URL so we can also add other system information later like CPU/RAM usage (as an example, not saying we should). Maybe `/System/Info/Resources` is a good path, as we already have the /System/Info part for 2 other endpoints?

```suggestion
    [HttpGet("Info/Resources")]
```

Reply:
hmm i really dont know. I would like to see those as seperate endpoints but then thinking this further this is not the most optimal api for other system resources like if you want CPU/RAM resources you would likely pull that every secound or so which is total overkill for the storage resources which will not change meaningfully in that timeframe but cost quite a bit of resources to evaluate.

I would be on board with `/System/Info/Storage` though

Reply:
Sounds good, let's do `GetSystemStorage` at `/System/Info/Storage` then 👍